### ISSUE TITLE
BiLSTM in the code

### ISSUE THREAD

**Issue by FantDing** [eragonruan/text-detection-ctpn#107]
Hi! I read the code recently and confused about some lines :joy: 
the `rpn_conv/3*3`  feature map is reshaped liike this: `img = tf.reshape(img, [N * H, W, C])`, dose it mean  the input `x(t)` is 256-d vector  at each timestep, but not 3*3*C d-nim descrided in paper
![image](https://user-images.githubusercontent.com/21337200/37332364-72bcfb38-26e1-11e8-87e3-fdc7a3eed933.png)

**Comment by eragonruan** [comment]
@FantDing yes, I did not implement this part. which means the sliding windows size is 1*1.
